packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -589,7 +589,7 @@ sdk.server('https://eu.api.example.com/v14');`)
         this.methodGenerics.get(operation.method).addOverload({
           typeParameters,
           parameters: [
-            { name: 'path', type: 'string' },
+            { name: 'path', type: `'${operation.path}'` },
```

Comment:
This is gross but if `type` is set to only `operation.path` then the generated TS output will be `get(path: /some/path)`, which obviously isn't valid. `type` here needs to literally be a string and yes `operation.path` is already a string and yes I also tried casting `operation.path` with `String(operation.path)`.

Just one of those weird `ts-morph` quirks.